Makefile

```diff
@@ -56,3 +55,11 @@ docker-cross: docker-build
 		"rm -f build && ln -s /tmp/build/ build && make cross"
 
 
+.PHONY: localstack
+localstack:
+	docker network create localstack
+	docker run --name localstack --network localstack --rm -d -p 4566:4566 -p 4571:4571 localstack/localstack
+
+.PHONY: docker
+docker:
+	docker build -t ghcr.io/dcoker/biscuit .
```

Comment:
we could actually run `make release`. The big difference though is that this can be done independently and has `bash` included (which we won't include in the release to keep the docker image small). This can maybe be consolidated in a later phase.